mne/io/eeglab/eeglab.py

```diff
@@ -371,6 +561,13 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
                 n_chan, n_times = [1, eeg.data.shape[0]]
             else:
                 n_chan, n_times = eeg.data.shape
+
+            # Seem to have transpose with matlab hdf storage
```

Comment:
MATLAB is column-major Python is row-major so this is expected

This should happen in the h5py branch in `_get_eeg_data`